.bit/responses/4.2-Week 4 Step 2.md

```diff
@@ -1,22 +1,68 @@
 ---
-files: 
-stepType: 
-scripts: 
+files: n/a
+stepType: checks
+scripts: 4.2.spec.js
 week: 4
 step: 2
 name: Week 4 Step 2
 ---
+Week 4 Step 2 ⬤⬤◯◯◯◯ | 🕐 Estimated completion: 20-25 minutes
 
-## Week 4 Step 2
+## Error Handling ~ Sir this is a Wendy's
+Ideally, users will fill the web form with the correct information. However, people make mistakes. In this step, we will be implementing some form validation to make sure that a user has inputted text value before submitting their password. 
 
-### Submitting an MVP
+### ✅  Task:
+- [ ]  Check you are on the `week4` branch 
+- [ ] Change your HTML form to accept `multipart/form-data` in `index.html`
+- [ ] Add a file input to `index.html` that accepts `image/x-png,image/jpeg` 
+- [ ]  Modify the  `getImage` event handler function in `index.js` to create an alert if either the file input or the `name` input is null
+- [ ] Commit your code to `bunnimage/index.js` and `bunnimage/index.html`
 
-A [minimum viable product](https://en.wikipedia.org/wiki/Minimum_viable_product) (MVP) is a version of a product with just enough features to be usable. Releasing an MVP means that developers potentially avoid lengthy and (ultimately) unnecessary work.
+### 🚧 Test your Work
+Open up your LiveServer plugin and start your server.
 
-Submit a working (loose definition) version of your MVP. Your project does not have to work smoothly or look presentable; it just needs to have a few main components (Azure Function included) that are interacting with each other.
+<details>
+<summary>:pencil: **You have three test cases to try:**</summary>
+  </br>
 
-### Getting Started
+1. **The "correct" way**: Submit both a file and text. Check that you receive "Thanks!"
+2. **The "unexpected" way (file)**: Submit a file that is not png or jpeg. Does it work?
+3. **The "unexpected" way (text input)**: Try submitting without entering a username. You should get an alert box that says "No name error."
+  <br><br/>
+</details>
 
-Starting and building a project from scratch can be daunting. Where do you even begin? Start by considering the project's bare bones. What does the program do at its core, without any aesthetic features or added functionality? In other words, what is the minimum viable product? Once you have that, break it down further. The key to building a project efficiently is to program one small part at a time and test each part as you go. For example, if your project has a webpage that takes user input, instead of creating all the components at once, make just a submit button and test it before moving on to the next component. Keep up the process of continuously creating and testing one piece at a time throughout the project. This will help save time on testing and debugging, as well as make the process less overwhelming. 
 
-**To move on, commit your MVP to this repo, and comment a brief summary of the problems you ran into while building it!**
+## Accepting Images as an Input
```

Comment:
The sections for the next two h2 headers can be placed under drop downs in case students want to challenge themselves to do it alone